### ISSUE TITLE
Error - On  .Net core

### ISSUE THREAD

**Issue by rmanjusha** [dotnet/BenchmarkDotNet#1405]
[error-benchmark.txt](https://github.com/dotnet/BenchmarkDotNet/files/4384481/error-benchmark.txt)

Hi,

I`m using benchmarkdotnet for first time and running on .net core 3.1 console application . While running the application is throwing exception as in attached file.

The build folder is '79925369-ddcb-4cb8-84a0-88dfabc0d345' is created on running benchmark project , but is getting deleted before the execution completes. Please help.

Thanks,
Manjusha

**Comment by adamsitnik** [comment]
Hello @rmanjusha 

We try to build the project two times, the failure in your log is from the second attempt and it fails with `file in use`. Most probably the first attempt failed with some other reason, but we don't know it.

Could you please follow our [troubleshooting](https://benchmarkdotnet.org/articles/guides/troubleshooting.html) guide?

TL;DR:

1.  Apply `[KeepBencharkFiles]` attribute to the class that defines benchmarks to make sure BDN is not removing the files
2. Run the benchmarks
3. Go to the folder with auto-generated files
4. Run `dotnet build` and copy-paste the output here.

Thanks,
Adam

**Comment by rmanjusha** [comment]
Hi Adam @adamsitnik 

I found the out the cause. The issue was with path length. I moved my project to root directory and is working fine now.

Thanks for quick response.

Thanks,
Manjusha
